packages/nx/src/plugins/package-json/create-nodes.ts

```diff
@@ -138,6 +138,7 @@ export function createNodeFromPackageJson(
   const hash = hashObject({
     ...json,
     root: projectRoot,
+    bust: 1,
```

Comment:
After this PR, with the same file content, a different result is obtained, so we need to bust the cache to ensure projects are processed.

Reply:
There should be a comment to explain why this is here and when to update it.